controllers/storagecluster/cephconfig.go

```diff
@@ -37,6 +38,18 @@ mon_pg_warn_max_object_skew = 0
 mon_data_avail_warn = 15
 [osd]
 osd_memory_target_cgroup_limit_ratio = 0.8
+[client.rbd-mirror.a]
+debug_ms = 1
+debug_rbd = 20
+debug_rbd_mirror = 30
+log_file = /var/log/ceph/$cluster-$name.log
+[client.rbd-mirror-peer]
+debug_ms = 1
+debug_rbd = 20
+debug_rbd_mirror = 30
+log_file = /var/log/ceph/$cluster-$name.log
+[mgr]
+mgr/rbd_support/log_level = debug
```

Comment:
Did you verify that these settings take effect -- `mgr/rbd_support/log_level` in particular?  In the BZ @travisn said that `defaultRookConfigData` doesn't currently use `ceph config assimilate-conf`, so this is not going to work.

Reply:
The following was suggested instead:

> [...] a setting in the mon store can be applied in Rook inside the postMonStartupActions() method [1] (only in the downstream branch such as 4.11).
> The code would look like: 
>```
> if err := monStore.Set("mgr", "mgr/rbd_support/log_level", "debug"); err != nil {>
>  logger.Errorf("failed to set rbd log level to debug. %v", err)
> }
>```
> [1] https://github.com/red-hat-storage/rook/blob/release-4.11/pkg/operator/ceph/cluster/cluster.go#L489